bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -204,11 +204,10 @@ pub mod pallet {
 
 			let dest = T::BridgeHubLocation::get();
 			let call =
-				Self::build_register_token_call(&origin_location, &asset_location, metadata)?;
+				Self::build_register_token_call(origin_location.clone(), asset_location, metadata)?;
```

Comment:
It seems you prefer using values over pointers. just curious what's the benefit of that?

I’d assume a pointer without cloning would be cheaper?

Reply:
if you look down the call stack, this actually avoids further cloning (clones once here instead of cloning twice later)